msdk-id-sirius/src/main/java/io/github/msdk/id/sirius/ConstraintsGenerator.java

```diff
@@ -79,7 +79,7 @@ public FormulaConstraints generateConstraint(MolecularFormulaRange range) {
       if (max!= maxNumberOfOneElements) constraints.setUpperbound(element, max);
     }
 
-    logger.debug("ConstraintsGenerator finished");
+    logger.debug("io.github.msdk.id.sirius.ConstraintsGenerator finished");
```

Comment:
It seems like the package name was added to more places than it was intended?

Reply:
Oh sorry, autorefactor by IDEA and I forgot to check this

Reply:
When you do auto-refactoring IDEA splits the refactorings into categories, like "actual code changes", "changes in comments", "uncategorized text changes" etc. Always take a look at everything that's not code, or just straight reject those (right click, exclude)